examples/DKIM_gen_keys.phps

```diff
@@ -13,7 +13,7 @@
  *
  * 2048 bits is the recommended minimum key length - gmail won't accept less than 1024 bits.
  * To test your DKIM config, use Port25's DKIM tester:
- * https://www.port25.com/support/authentication-center/email-verification/
+ * https://tools.sparkpost.com/dkim
```

Comment:
Good fix - though in that case the other references to Port25 should also be updated to say Sparkpost too.

Reply:
Ok
Can I only edit this link

Reply:
No, you can make any changes you like, including reverting things. You can either mark resolved things that you're sure about, or you can leave that to me.

Reply:
How can I leave it to you
Should I click resolve conversation

Reply:
How do I leave it you
Should I click resolve conversation

Reply:
I mean you can update Port25 to Sparkpost in the text as well as the URL. So change the line above to say "use Sparkpost's DKIM tester". There's another mention of Port25 that's probably outdated too.